packages/styles/src/components/form-check.scss

```diff
@@ -158,7 +158,7 @@
       border: form-check.$form-check-input-border-width solid CanvasText;
 
       &:checked {
-        background-color: CanvasText;
+        background-color: SelectedItem;
```

Comment:
@alizedebray this is what I was looking for. It makes not difference in Firefox though with Highlight color, but it does make one in blink-based browser.